ironfish-rust-nodejs/src/frost.rs

```diff
@@ -28,6 +32,35 @@ pub struct NativeCommitment {
     pub binding: String,
 }
 
+pub struct IdentifierSignatureShare {
+    pub identifier: Identifier,
+    pub signature_share: SignatureShare,
+}
+
```

Comment:
Starting the migration to `SignatureShare` from `SigningShare`, we can fixup the rest in a following PR.